mbuild/tests/test_lammpsdata.py

```diff
@@ -456,65 +493,173 @@ def test_lj_pairs(self, ethane):
                     assert np.isclose(sigma, 1.00)
                     checked_section = True
 
-    def test_lj_bonds(self, ethane):
-        from foyer import Forcefield
-
-        OPLSAA = Forcefield(name="oplsaa")
-        structure = OPLSAA.apply(ethane)
-        write_lammpsdata(
-            filename="lj.lammps", structure=structure, unit_style="lj"
+    def test_lj_passed_pairs(self, ethane, lj_save):
+        fn = lj_save(
+            ethane,
+            Path.cwd(),
+            sigma=1,
+            epsilon=1,
         )
+        checked_section = False
+        with open(fn, "r") as fi:
+            while not checked_section:
+                line = fi.readline()
+                if "Pair Coeffs" in line:
+                    fi.readline()
+                    line = fi.readline().split()
+                    epsilon = float(line[1])
+                    sigma = float(line[2])
+                    assert np.isclose(epsilon, 0.276144, atol=1e-5)
+                    assert np.isclose(sigma, 0.35)
+                    checked_section = True
 
+    def test_real_pairs(self, ethane, real_save):
+        fn = real_save(ethane, Path.cwd())
         checked_section = False
-        with open("lj.lammps", "r") as fi:
+        with open(fn, "r") as fi:
             while not checked_section:
                 line = fi.readline()
-                if "Bond Coeffs" in line:
+                if "Pair Coeffs" in line:
                     fi.readline()
-                    bonds = list()
-                    bonds.append(float(fi.readline().split()[1]))
-                    bonds.append(float(fi.readline().split()[1]))
-                    assert np.allclose(sorted(bonds), [49742.424, 63106.06])
+                    line = fi.readline().split()
+                    epsilon = float(line[1])
+                    sigma = float(line[2])
+                    assert np.isclose(epsilon, 0.066)
+                    assert np.isclose(sigma, 3.5)
                     checked_section = True
 
-    def test_lj_angles(self, ethane):
-        from foyer import Forcefield
+    def test_lj_bonds(self, ethane, lj_save):
+        fn = lj_save(ethane, Path.cwd())
+        checked_section = False
+        ethane_bondsk = (
+            np.array([224262.4, 284512.0]) * (0.35) ** 2 / (0.276144) / 2
```

Comment:
I see that you are accounting for k/2 here in your test case. This needs to also be addressed in lammpsdata.py

Reply:
But the tests passes! So clearly it's taking things into account. I believe this happens in the foyer xml conversion to Parmed. Again, sit down and do this by pen and paper if you have to, but that's what I did and it looks sensible to me. And these tests should catch the writer if it's writing the wrong values.

Reply:
See my comment (below). It may work for ethane (since mbuild and foyer xml are specified in real units), but let's see if it works for a case where both are specified in reduced units. 

There will be cases where the user will not want to convert from real to reduced and would prefer to have everything consistently in reduced units.

Reply:
I think this is addressed, I got the correct output using the CLP.xml file you provided below so I think we're good to go. I'll leave this comment open though just in case you guys catch something else.